src/core/util/util.spec.js

```diff
@@ -443,6 +443,22 @@ describe('util', function() {
       flush(function(){ expect( callBackUsed ).toBeUndefined(); });
     }));
 
+    it('should only skip callbacks of scopes which were destroyed', inject(function($mdUtil) {
+      var callBackUsed1, callback1 = function(){ callBackUsed1 = true};
```

Comment:
FWIW, I like the way the callbacks are tracked via a jasmine spy (see line 232), but I used the same idiom used in the previous test since these tests are related.  I'm happy to update both tests to use the spy idiom if you'd like.